pom.xml

```diff
@@ -77,6 +77,16 @@
       <groupId>co.cask.cdap</groupId>
       <artifactId>cdap-formats</artifactId>
       <version>${cdap.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.apache.avro</groupId>
+          <artifactId>avro</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>io.thekraken</groupId>
```

Comment:
why do we need this change?

Reply:
we don't need to exclude it, but it reduces the size of the jar. This dependency will be removed from CDAP when stream formats are removed.